scripts/inc.packagetarget.sh

```diff
@@ -85,6 +85,21 @@ createzip() {
   find "$build" -exec touch -d "2008-02-28 21:33:46.000000000 +0100" {} \;
   cd "$build"
 
+  MEMORY_MIN=800000 # Minimum of RAM required (for single thread) on x86_64 machine [~701MB for xz, 2*25KB for bash and some spare]
+  THREADS="$(($(cat /proc/cpuinfo | grep "^processor" | wc -l)))"
+
+  if ! grep -q "MemAvailable:" /proc/meminfo; then
+    MEMORY=0
+  else
+    MEMORY="$(($(grep "MemAvailable:" /proc/meminfo | awk '{print $2}') / 4))"
+  fi
+
+  if [ $MEMORY = 0 ] || [ $MEMORY -lt $MEMORY_MIN ]; then
```

Comment:
Actually I think I encountered this in one of those 3 :-)
I guess older bash and/or busbox's ash built with some option.